op-txproxy/go.mod

```
@@ -0,0 +1,92 @@
+module github.com/ethereum-optimism/optimism/op-txproxy
```

Please update the module 
```suggestion
module github.com/ethereum-optimism/infra/op-txproxy
```

---

ahh nice catch, had this service in the monorepo initially